cmd/activator/main.go

```diff
@@ -135,14 +136,26 @@ func (a *activationHandler) handler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, msg, int(status))
 		return
 	}
+
+	var transport http.RoundTripper
+	if endpoint.IsVerified() {
```

Comment:
Not clear as to why the retry approach is better when the readinessProbe is defined by the user. The readinessProbe is a clear indicator of when the app is ready to receive requests.

Reply:
For GETs a request could still fail for whatever reason even if readiness succeeds

Reply:
Once the user's application says it's ready, it will be put into service.  That's true whether it's the first pod or the 100th.  The only reason we're retrying at this level (in the activator) is because we know the network programming is eventually consistent, which matters more for the first pod.

Once we've verified we can reach the service with readiness probing, we should just forward the request.  If it fails because of something in the user's application, we should just rely on a higher level retry (or not).